### ISSUE TITLE
Crawler failing with <unspecified file>(1): expected value

### ISSUE THREAD

**Issue by jrymart** [bencabrera/grawitas#14]
When I try to run the crawler on a text file it prints all of the article names, and then fails with: 
ABORTING. The application terminated with an exception:
<unspecified file>(1): expected value

At first I thought it was because I was doing it with lists of Wikipedia:Arbitration pages, and thought I was running into a similar issue as was had with the user talk pages being labeled slightly differently, but I tried it with a list of all articles (attached) and got the same error.  Any insight would be much appreciated! 

Attached is the file I was trying to crawl
[watched_pages.txt](https://github.com/bencabrera/grawitas/files/2527186/watched_pages.txt)

**Comment by bencabrera** [comment]
Hi,

sorry for the delay...

I was able to reproduce the error. It seems this could be an encoding issue. When I checked your file encoding via `file -bi watched_pages_copy.txt` I got `text/plain; charset=unknown-8bit`. However, the programs are only able to work on UTF-8 encoded files (resp. standard ascii files). So I replaced the Carriage returns  (dez 13, hex 0D in ascii) in your file with newlines using `sed -i -e 's/\x0D/\n/g' watched_pages_copy.txt`. 
It then seemed to work (obviously I did not let it run through). I attached the file 
[watched_pages_ascii.txt](https://github.com/bencabrera/grawitas/files/2547935/watched_pages_copy.txt)

So make sure you have UTF-8 encoded input files. 

Let me know if this works for you, then we can close the issue.
